ioctl/newcmd/config/config_reset.go

```diff
@@ -5,3 +5,28 @@
 // License 2.0 that can be found in the LICENSE file.
 
 package config
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl"
+)
+
+// NewConfigReset resets the config to the default values
+func NewConfigReset(client ioctl.Client, defaultConfigFileName string) *cobra.Command {
```

Comment:
delete argument because user can't know  file name and path of `defaultConfigFileName`.

Reply:
you can get it from client.  
1. add client.ConfigFilePath()  in client.go  
2. call it in line23